packages/@aws-cdk/aws-servicecatalog/test/integ.product.ts

```diff
@@ -0,0 +1,23 @@
+import * as path from 'path';
+import * as cdk from '@aws-cdk/core';
+import * as servicecatalog from '../lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'integ-servicecatalog-product');
+
+new servicecatalog.CloudFormationProduct(stack, 'TestProduct', {
+  productName: 'testProduct',
+  owner: 'testOwner',
+  cloudFormationProductVersions: [
+    {
+      cloudFormationTemplate: servicecatalog.CloudFormationTemplate.fromUrl(
+        'https://awsdocs.s3.amazonaws.com/servicecatalog/development-environment.template'),
+    },
+    {
+      validateTemplate: false,
```

Comment:
Hmm. Why is this `false` here?

Reply:
I left one not set(default to true) and the other set to false so both are tested.

Reply:
Can you flip it around, so that the one dowloaded from the internet is not validated?